Source/Core/DiscIO/WbfsBlob.h

```diff
@@ -19,7 +19,10 @@ class WbfsFileReader : public IBlobReader
 public:
 	static WbfsFileReader* Create(const std::string& filename);
 
-	u64 GetDataSize() const override { return m_size; }
+	// The WBFS format does not save the original file size.
+	// This function returns a constant upper bound
+	// (the size of a double-layer Wii disc).
```

Comment:
What about WBFSs of single layer disks?

Would it make sense to check m_wlba_table to see if there are any entries in the second layer and return the single layer size if none are found?

Not that md5ing a WBFS volume makes sense to start with, as it's a lossy disk compression format.